src/Features/Quotas/TeamQuotasOverview/TeamQuotasOverview.tsx

```diff
@@ -36,7 +30,9 @@ export function TeamQuotasOverview({ teams }) {
   const teamData = teams.find((team) => team.id === teamId);
 
   if (error)
-    return <Error404 header="Team not found" title="Crikey. We can't find the team you are looking for." message="" />;
+    return (
+      <EmptyState header="Team not found" title="Crikey. We can't find the team you are looking for." message="" />
```

Comment:
Maybe we should make this consistent with the other error messages and use a title and a message instead of header and message?